src/experimental/build/utils/dfx.ts

```diff
@@ -0,0 +1,210 @@
+import { HttpAgent } from '@dfinity/agent';
+import { Secp256k1KeyIdentity } from '@dfinity/identity-secp256k1';
+import { execSync } from 'child_process';
+
```

Comment:
Make an issue about finishing the work in the experimental milestone, finishing the work of adding the `--network` option to all commands

Reply:
https://github.com/demergent-labs/azle/issues/2932